addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -339,18 +339,29 @@ var SnippetEditor = Widget.extend({
             return;
         }
 
+        const {transform} = targetEl.style;
+        if (transform) {
+            targetEl.style.transform = '';
+        }
+
         // Now cover the element
         const offset = $target.offset();
         var manipulatorOffset = this.$el.parent().offset();
         offset.top -= manipulatorOffset.top;
         offset.left -= manipulatorOffset.left;
         this.$el.css({
             width: $target.outerWidth(),
+            height: $target.outerHeight(),
```

Comment:
How was the height handled before? CSS I guess? But I don't see it adapted by your commit?